daisy_workflows/image_import/inspection/src/boot_inspect/model.py

```diff
@@ -36,6 +36,8 @@ class Distro(enum.Enum):
   SLES = 'sles'
   UBUNTU = 'ubuntu'
   WINDOWS = 'windows'
+  WINDOWS_DESKTOP = 'windows-desktop'
+  WINDOWS_SERVER = 'windows-server'
```

Comment:
Doesn't WINDOWS include the other two?

Reply:
You're right, good catch -- that was a previous iteration that I intended to remove.

Removed WINDOWS_DESKTOP and WINDOWS_SERVER